consensus/polybft/state_sync_manager.go

```diff
@@ -362,14 +362,18 @@ func (s *stateSyncManager) PostEpoch(req *PostEpochRequest) error {
 }
 
 type PostBlockRequest struct {
-	// Block is a reference of the executed block
-	Block *types.Block
+	// FullBlock is a reference of the executed block
```

Comment:
Early comment, can you move these functions (i.e. `PostX` to a new `handlers.go` file? Now they are going to be generic for different modules.

I am open to a better name. We can leave that for the PR review.

Reply:
Sure. New file for these structs is definitely better.